app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/MessageCardViewHolder.kt

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.sessioncontrol.viewholders.onboarding
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import org.mozilla.fenix.R
+import org.mozilla.fenix.databinding.NimbusMessageCardBinding
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.ext.increaseTapArea
+import org.mozilla.fenix.gleanplum.Message
+import org.mozilla.fenix.gleanplum.MessagesManager
+import org.mozilla.fenix.home.sessioncontrol.SessionControlInteractor
+
+class MessageCardViewHolder(
+    view: View,
+    private val interactor: SessionControlInteractor
+) : RecyclerView.ViewHolder(view) {
+    private val messagesManager: MessagesManager = view.context.components.messagesManager
+    private val message: Message? = messagesManager.getNextMessage()
+
+    init {
+        initialize(view)
+    }
+
+    private fun initialize(view: View) {
+        val binding = NimbusMessageCardBinding.bind(view)
+        val safeMessage = message ?: return
+
+        binding.messageButton.setOnClickListener {
+            interactor.onMessageClicked(safeMessage)
+
+        }
+
+        binding.descriptionText.text = safeMessage.data.text
+        // TODO update with the right info.
```

Comment:
@jhugman could you give me a hand? :) I'm still having difficulties bringing the  button from fml, maybe we could pair on it.
`button-label` same as https://github.com/mozilla-mobile/fenix/pull/24048#discussion_r819986757